src/Game.java

```diff
@@ -0,0 +1,91 @@
+import java.io.FileNotFoundException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+/**
+ * Created by EddyJ on 6/21/16.
+ */
+public class Game {
+    private Player player1;
+    private Player player2;
+    private int sticks;
+    private Player currentPlayer = null;
+    String loser = null;
```

Comment:
`winner` and `loser` should probably simply be set to player objects.  At this point the only way to get which player won for a game(getting the object) would be to loop through the players and compare the name in the object.  Additionally, while it might not be intuitive this way actually uses up MORE memory because instead of just pointing to the object it is a whole new String object.